modelcenter/PP-LCNetV2/introduction_en.ipynb

```diff
@@ -0,0 +1,239 @@
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 4. Theory"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 4.1 Overall Structure\n",
+    "\n",
+    "The overall structure of the network is shown in the figure below:\n",
+    "\n",
+    "<div align=\"center\">\n",
+    "<img src=\"https://user-images.githubusercontent.com/12560511/200776415-19f7f09d-f8e9-4b64-bbcd-72ac418c23cf.png\"  width = \"80%\"  />\n",
+    "</div>\n",
+    "\n",
+    "### 4.2 Model Details\n",
+    "\n",
+    "PP-LCNetV2 model is optimized on the basis of PP-LCNetV1. It mainly uses the reparameterization strategy to combine the deep convolution of convolution cores of different sizes, and optimize the point convolution and Shortcut, etc.\n",
+    "\n",
+    "#### 4.2.1  Re-parameterization\n",
+    "\n",
+    "The size of the convolution kernel affects the size of the model's receptive field, which affects the model's ability to capture more global or local features. In order to help the model build different scales features, we use 5*5, 3*3 and 1*1 size convolution kernel. The details is shown in the figure below.\n",
+    "\n",
+    "<div align=\"center\">\n",
+    "<img src=\"https://user-images.githubusercontent.com/12560511/201664156-be8f5dc6-5345-436a-962e-079139f378d5.png\"  width = \"80%\"  />\n",
+    "</div>\n",
+    "\n",
+    "#### 4.2.2 PW Conv\n",
+    "\n",
+    "The depth separable convolution usually consists of a layer of DW convolution and a layer of PW convolution to replace the standard convolution. In order to make the depth separable convolution have a stronger fitting ability, we try to use two layers of PW convolution. Meanwhile, in order to control the efficiency of the model, the two-layer PW convolution is set as: The first is to compress the feature map in the channel dimension, and the second is to zoom in to restore the feature map channel, as shown in the figure below. Experiments show that this strategy can significantly improve model performance. Meanwhile, in order to balance the impact on model efficiency, PPLCNetV2 only uses this strategy in Stage4.\n",
+    "\n",
+    "<div align=\"center\">\n",
+    "<img src=\"https://user-images.githubusercontent.com/12560511/201664376-e3cdd4ac-0767-4773-a6cc-b6885796bc1e.png\"  width = \"80%\"  />\n",
+    "</div>\n",
+    "\n",
+    "#### 4.2.3 Shortcut\n",
+    "\n",
+    "The residual structure has been widely used by many models since it was proposed, but in lightweight convolutional neural networks, the elite-wise addition operation brought by the residual structure will affect the speed of the model. In PP-LCNetV2, The effect of the residual structure on the model was tested with the Stage as the unit, and it was found that the use of the residual structure did not necessarily improve the performance, so PPLCNetV2 only used the residual structure in the last Stage: add Shortcut in the Block, as shown in the figure below.\n",
```

Comment:
elite-wise -> elementwise

Reply:
The residual structure has been widely used by many models since it was proposed, but in lightweight convolutional neural networks, the elementwise addition operation brought by the residual structure will affect the speed of the model. We experimented on the influence of shortcut on the model at different stage. Finally, we only used Shortcut in the last block, as shown in the figure below.